Configuration/Test/test.py

```diff
@@ -0,0 +1,534 @@
+from SimG4Core.Application.hectorParameter_cfi import *
+import FWCore.ParameterSet.Config as cms
+import copy
+
+#
+# 09.08.17 patrykel
+# used to test new optics in 8_1_0
+#
+
+
+process = cms.Process("TestFlatGun")
+
+'''process.MessageLogger = cms.Service("MessageLogger",
+    destinations = cms.untracked.vstring('warnings',
+        'errors',
+        'infos',
+        'debugs'),
+    categories = cms.untracked.vstring('ForwardSim',
+        'TotemRP'),
+    debugModules = cms.untracked.vstring('*'),
+    errors = cms.untracked.PSet(
+        threshold = cms.untracked.string('ERROR')
+    ),
+    warnings = cms.untracked.PSet(
+        threshold = cms.untracked.string('WARNING')
+    ),
+    infos = cms.untracked.PSet(
+        threshold = cms.untracked.string('INFO')
+    ),
+    debugs = cms.untracked.PSet(
+        threshold = cms.untracked.string('DEBUG'),
+        INFO = cms.untracked.PSet(
+            limit = cms.untracked.int32(0)
+        ),
+        DEBUG = cms.untracked.PSet(
+            limit = cms.untracked.int32(0)
+        ),
+        TotemRP = cms.untracked.PSet(
+            limit = cms.untracked.int32(1000000)
+        ),
+        ForwardSim = cms.untracked.PSet(
+            limit = cms.untracked.int32(1000000)
+        )
+    )
+)'''
```

Comment:
Is this needed ?

Reply:
Just for debugging, so for now it is not necessary.
Will remove it.

Reply:
do it